### DISCUSSION TITLE
Is it possible to reuse a mapper in other mappers automagically?

### DISCUSSION THREAD

**Discussion by salmanbabri** [riok/mapperly#1542] [Q&A] [answered]
Example I've a mapper to convert Customer to CustomerDto:-

```
public static class CustomerMapper
{
    public static partial CustomerLite ToCustomerLite(Customer customer);
}
```

Now I want this to be reused when another object's mapper requires to convert Customer to CustomerLite. Eg when converting Invoice to InvoiceDetailResponse, I want invoice.Customer to be translated to CustomerLite using already declared method above.

```
public static class InvoiceMapper
{
    public static partial InvoiceResponse ToInvoiceResponse(Invoice invoice);
}
```

**Comment by latonz** [comment, marked as answer]
Have you tried using `UseStaticMapper` / `UseMapper`? See [docs](https://mapperly.riok.app/docs/configuration/user-implemented-methods/#use-external-mappings).
